tests/test_simpleButler.py

```diff
@@ -49,9 +49,11 @@ class SimpleButlerTestCase(unittest.TestCase):
     can instead utilize an in-memory SQLite Registry and a mocked Datastore.
     """
 
-    datasetsManager = "lsst.daf.butler.registry.datasets.byDimensions.ByDimensionsDatasetRecordStorageManager"
-    datasetsImportFile = "datasets.yaml"
```

Comment:
Does this mean that this file is no longer used?

Reply:
I've left in place the test that imports this file (with integer IDs) into a UUID data repository.  I don't know if we want to support that indefinitely, but I haven't tried to deprecate it (I didn't think about it at all when writing the RFC).